docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -28,9 +28,9 @@ public static void Example()
                 "images");
 
             //Download the image set and unzip
-            string finalImagesFolderName = DownloadImageSet(
-                imagesDownloadFolderPath);
-
+            //string finalImagesFolderName = DownloadImageSet(
+            //    imagesDownloadFolderPath);
```

Comment:
Delete the comment. The `DownloadImageSet` method should also be deleted if it is not used anymore.

Reply:
Why do you feel DownloadImageSet is not used anymore? We are using it to download images that we will train on. This change merely changes the input type for images in imageclassification API from string to VBuffer<bytes> so that user can do inferencing on in-memory images. 

---
In reply to: [327499153](https://github.com/dotnet/machinelearning/pull/4242#discussion_r327499153) [](ancestors = 327499153)

Reply:
That's fine, of course you should not delete it if it is used. In the iteration I was looking at I didn't notice where it was being called, that's why I was asking.

---
In reply to: [329333913](https://github.com/dotnet/machinelearning/pull/4242#discussion_r329333913) [](ancestors = 329333913,327499153)